installers/installer.sh

```diff
@@ -544,7 +531,7 @@ function __install_couchbase() {
         fi
         __install_couchbase_debian "$version" "$tmp"
     elif [[ "$os" == "RHEL" ]]; then
-        if [[ "$OS_VERSION" == "8" ]]; then
+        if [[ "$OS_VERSION" == "8" ]] || [[ "$OS_VERSION" == "8.1" ]]; then
```

Comment:
Can you just do major version?

Reply:
This isn't even needed anymore.  I actually only take the major version now.